recipes/nf-test/meta.yaml

```diff
@@ -0,0 +1,33 @@
+{% set version = "0.6.0" %}
+
+
+package:
+  name: "nf-test"
+  version: {{ version }}
+
+build:
+  number: 0
+  noarch: generic
+
+source:
+  url: https://github.com/askimed/nf-test/releases/download/v{{version}}/nf-test-{{version}}.tar.gz
+  sha256: 5dd9a4b5f370eefbeb04cf5dc397e8c998337017b4f5f77a6b719ec438dc3f59
+
+requirements:
+  host:
+    - openjdk >=11,<=18
+    - coreutils
+    - curl
+  run:
+    - openjdk >=11,<=18
+    - coreutils
+    - curl
+
+test:
+  commands:
+    - nf-test version
+
+about:
+  home: 'https://code.askimed.com/nf-test/'
```

Comment:
I think GitHub URLs could be here for source and issues and stuff off the top of my head? But it's minor.